test/Token.t.sol

```diff
@@ -363,23 +363,23 @@ contract TokenTest is NounsBuilderTest, TokenTypesV1 {
         assertEq(token.getPastVotes(founder, block.timestamp - 1), 2);
     }
 
-    function testRevert_OnlyAuctionCanMint() public {
```

Comment:
Let's harden this test a bit, and turn it in to two. 

Firstly, let's make a separate test that the auction is a minter after deployment. That should cover the default case.

Secondly, let's make this a [fuzz test](https://book.getfoundry.sh/forge/fuzz-testing) that takes an address, updates the minter, and tries to mint.